tests/eval.nix

```diff
@@ -1,5 +1,8 @@
 {
   int = 123;
+  bin = 0b1111011;
```

Comment:
```suggestion
  int2 = 0123;
  bin = 0b1111011;
```

Reply:
It's an extra test case, not really a rename. It's unfortunate that github renders it like that.